### ISSUE TITLE
Make error

### ISSUE THREAD

**Issue by zovelsanj** [MP-Gadget/MP-Gadget#625]
I am trying to build MP-Gadget but was frequently getting this error.

```
mpicc -fopenmp -O3 -g -Wall  -Zp16 -fp-model fast=1 -simd -fPIC -o bigfile-mpi.o -c bigfile-mpi.c
gcc: error: unrecognized command-line option ‘-Zp16’
gcc: error: unrecognized command-line option ‘-fp-model’; did you mean ‘-fipa-modref’?
gcc: error: unrecognized command-line option ‘-simd’
make[2]: *** [Makefile:25: bigfile-mpi.o] Error 1
```
My system is Linux. I followed the instructions in Readme. I also installed `openmpi` with `apt` and built `bigfile` from source at `depends/bigfile` successfully.  `mpicc` works on `c` files but here it is not recognizing these specific command-line options.

I `make` the Makefile at `depends/bigfile/src` which resolved this error to some extent, but now when i `make` the main Makefile`mpicc` isn't able to generate executables.
```
configure: error: in `/home/graviton/Documents/Learning/Astrophysics/MP-Gadget/depends/tmp-pfft-1.0.8-alpha3-fftw3-2don2d/double':
configure: error: C compiler cannot create executables
See `config.log' for more details
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for style of include used by make... GNU
checking for gcc... mpicc
checking whether the C compiler works... no
make: *** [Makefile:21: pfft] Error 1
```

Here is the config-log file. 
[config.log](https://github.com/MP-Gadget/MP-Gadget/files/13681130/config.log)

**Comment by sbird** [comment]
Looks like you are using platform-options/Options.mk.stampede2, which is for an intel compiler. Try Options.mk.example instead and it should work better.

**Comment by zovelsanj** [comment]
> Looks like you are using platform-options/Options.mk.stampede2, which is for an intel compiler. Try Options.mk.example instead and it should work better.

Thank you Dr. @sbird. The problem was exactly what you pointed out. The `OPTIMIZE` flag of `Options.mk` and `platform-options/Options.mk.stempede2` were the same i.e.
`OPTIMIZE =  -fopenmp -O3 -g -Wall -Zp16 -xHost -fp-model fast=1`
I replaced that of `Options.mk` with 
`OPTIMIZE =  -fopenmp -O3 -g -Wall -ffast-math` 
and I was able to successfully build `MP-Gadget`. Thank you again sir.
